package.json

```diff
@@ -3,7 +3,7 @@
   "description": "React Native module for Radar, the leading geofencing and location tracking platform",
   "homepage": "https://radar.com",
   "license": "Apache-2.0",
-  "version": "3.8.4",
+  "version": "3.8.5",
```

Comment:
`npm install` to update package-lock.json

Reply:
Done